seqr/management/tests/deactivate_project_search_tests.py

```diff
@@ -31,7 +31,7 @@ def test_command(self, mock_logger, mock_input):
         # Test success
         mock_input.return_value = 'y'
         call_command('deactivate_project_search', PROJECT_GUID)
-        mock_logger.info.assert_called_with('Deactivated 14 samples')
+        mock_logger.info.assert_called_with('Deactivated 11 samples')
```

Comment:
this is a change in behavior, but a desired one - there is no need to reactivate RNA data in a project when we are deactivating search data, as they are unrelated